lib/model/profile_model.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:equatable/equatable.dart';
+import 'package:survey_flutter_ic/api/response/profile_response.dart';
+
+class ProfileModel extends Equatable {
+  final String avatarUrl;
+
```

Comment:
I think we need `id` as well 🙏

Reply:
@Wadeewee In Which case do we need to use the `id`? 🤔 Because I see it seems useless so I don't add it into the model 😄

Reply:
Sorry, I thought we have to use `id` for calling `SurveyDetail` but we can use the `id` from `Surveys` instead 🙏